Makefile

```diff
@@ -87,7 +87,7 @@ build:
 	make stop
 
 clean: deps
-	rm -rf $(GOPATH)/pkg/*
```

Comment:
Updated the commit to use a wildcard for the OS/arch directory... this comment didn't get collapsed because I didn't edit the specific line you commented on.